java-generic-mqtt/src/main/javadoc/overview.html

```diff
@@ -0,0 +1,83 @@
+<HTML>
```

Comment:
Can be improved?

Reply:
Javadoc should be generated automatically from java code, I would move this to just a simple `README.md`

Reply:
This is merely the example that when compiled, merges with JavaDoc.